routes/routes.go

```diff
@@ -0,0 +1,55 @@
+package routes
+
+import (
+	"github.com/gofiber/fiber"
+	"learning_go/book"
+	"learning_go/database"
+)
+
+func GetBooks(c *fiber.Ctx) {
+	db := database.DB.Db
+	var books []book.Book
+	db.Find(&books)
```

Comment:
is there any kind of error handling to do here if the lookup fails, or the database is suddenly unavailable during this call, etc.? Or will that other panic middleware catch all of that, do you think?